book3/14-objects.mkd

```diff
@@ -1,97 +1,100 @@
-Managing larger programs
+Manejando programas más grandes
 ------------------------
 
-\index{object-oriented}
+\index{orientada a objetos}
 
-At the beginning of this book, we came up with four basic programming
-patterns which we use to construct programs:
+Al comienzo de este libro, vimos cuatro patrones básicos de programación
+que utilizamos para construir programas:
 
-* Sequential code
-* Conditional code (if statements)
-* Repetitive code (loops)
-* Store and reuse (functions)
+* Código secuencial
+* Código condicional (declaraciones 'if')
+* Código repetitivo (bucles)
+* Almacenar y reutilizar (functiones)
 
-In later chapters, we explored simple variables as well as collection
-data structures like lists, tuples, and dictionaries.
+En capítulos posteriores, exploramos las variables simples, así como 
+estructuras de datos recolectados, tales como listas, tuplas y diccionarios.
 
-As we build programs, we design data structures and write code
-to manipulate those data structures.   There are many ways to write programs
-and by now, you probably have written some programs that are "not so elegant"
-and other programs that are "more elegant". Even though your programs
-may be small, you are starting to see how there is a bit of art and
-aesthetic to writing code.
+A medida que construimos programas, diseñamos estructuras de datos y 
+escribimos código para manipularlas. Hay muchas formas de escribir programas
+y, a estas alturas, probablemente hayas escrito algunos programas "no muy elegantes"
+y otros que son "más elegantes". Aunque tus programas aún
+sean pequeños, estás empezando a ver que al escribir código
+hay una parte de arte y de consideraciones estéticas.
 
-As programs get to be millions of lines long, it becomes increasingly important
-to write code that is easy to understand.  If you are working on a million-line
-program, you can never keep the entire program in your mind at the same time.
-We need ways to break large programs into multiple smaller pieces so that
-we have less to look at when solving a problem, fix a bug, or add a new feature.
+A medida que los programas crecen hasta abarcar millones de líneas de código, 
+se vuelve cada vez más importante que éste resulte fácil de entender. Si estás
+trabajando en un programa de un millón de líneas, es imposible mantener todo
+el programa en tu mente a la vez. Necesitamos maneras de dividir programas grandes
+en varias piezas más pequeñas para que tengamos que 
+concentrarnos en una sección menor cuando tengamos que resolver un problema,
+arreglar un bug, o agregar una nueva funcionalidad.
 
-In a way, object oriented programming is a way to arrange your code
-so that you can zoom into 50 lines of the code and understand it
-while ignoring the other 999,950 lines of code for the moment.
+En cierto modo, la programación orientada a objetos es una forma
+de ordenar tu código de tal manera que puedas enfocarte en 50
+líneas de código y entenderlas, e ignorar las otras 999.950
```

Comment:
Let's keep the comma here, 999.950 -> 999,950

Reply:
Ok, pero en castellano el divisor de números es el punto. Si dejamos la coma, estaría significando "novecientas noventa y nueve coma noventa y cinco", no "novecientas noventa y nueve mil novecientas cincuenta".

Reply:
De acuerdo, entonces hay que mantener el formato con el punto.